fossevents/events/views.py

```diff
@@ -14,3 +18,18 @@ def event_detail(request, pk, slug=None, template_name='events/event_detail.html
         'event': event,
     }
     return render(request, template_name, ctx)
+
+
+class EventCreateView(SendMailPostSaveMixin, CreateView):
+    form_class = EventForm
+    template_name = 'events/event_form.html'
+
+
+class EventUpdateView(SendMailPostSaveMixin, UpdateView):
```

Comment:
It's it sending the same email for both the action, when the event is added and when updated, but that shouldn't be the case. It's a update we need to be very clear that event is updated (ideally, with the diff), else no email please. This works solution works, but it will be quite annoying, esp. when it's sent to a group of moderators.